api/graphql/schema.graphql

```diff
@@ -1794,6 +1795,9 @@ type Mutation {
   archiveApplication(id: ID!): PoolCandidate
     @guard
     @can(ability: "archiveApplication", find: "id")
+  changeApplicationSuspendedAt(id: ID!, setSuspended: Boolean!): PoolCandidate
```

Comment:
Feel free to discard this suggestion if you don't agree. :smile:  What about changing the second parameter name to `isSuspended` since it is a boolean, not a DateTime like the actual database field.

Reply:
Changed with [2290b6a](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/2290b6ac9c081482788c0139c5347b6e99a16450).